packages/configs/tsc/index.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "@parcel/config-default",
+  "transforms": {
+    "*.{ts,tsx}": [
+      "@parcel/transformer-typescript-tsc",
+      "@parcel/transformer-js"
```

Comment:
This shouldn't be needed as part of the `.ts` config (and in fact will never run). The tsc transformer will output an asset of type `js`, which will cause it to jump to another pipeline and never run the rest of the ts pipeline.